.drone.yml

```diff
@@ -335,7 +335,7 @@ steps:
     image: 340268328991.dkr.ecr.eu-west-2.amazonaws.com/acp/trivy/client:latest
     pull: always
     environment:
-        IMAGE_NAME: coa:${DRONE_COMMIT_SHA}
+        IMAGE_NAME: :${DRONE_COMMIT_SHA}
```

Comment:
Not sure how the `IMAGE_NAME` is used later, but there is an extra colon, before there was a custom name `coa`, perhaps the `$IMAGE_REPO` is missing?